src/patternfly/upgrade-examples/LoginUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,12 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
+| -- | -- | -- | -- | -- |
+| `.login-pf` | `<div>` | `.pf-c-login` | `<div>` | Creates a login containing content. |
+| `.login-pf-heading` | `<div>` | `.pf-c-login__header` | `.pf-c-login` | Creates the header of a login. |    
```

Comment:
this looks like `.login-pf-page-header` on pf3, and it applies to `<header>`. `.pf-c-login__header` appplies to `<header>` in pf4, too, not `.pf-c-login`

Reply:
Class to element tag has been rematched.